src/pages/Question.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import Logo from '../components/Logo.js'
+import Header from '../components/Header.js'
+import  {questions} from '../config';
+import '../styles/questions.scss';
+
+const Question = () => {
+    const categories = ['food', 'sleep', 'sport', 'misc'];
+    let currentCategory = 0;
```

Comment:
dit kan beter in een State gezet worden, kijk hiervoor naar useState() : https://reactjs.org/docs/hooks-state.html